tests/4.5/target_enter_exit_data/test_target_enter_exit_data_depend.c

```diff
@@ -124,6 +125,7 @@ int test_async_between_target() {
   }
   errors = 2*N != sum;
 
+#pragma omp target exit data map(release: h_array[0:N], val)
```

Comment:
Nothing wrong in the suggestions, just curious why "release" and not "delete" here ?

Reply:
Either should work - one forces it, the other uses the ref count (which should go to 0). The idea was to have some variance in the hope that it might find some additional bugs (unlikely). — I can change it, if you have any preference.

Reply:
Not at all, I like variance ! I have already approved the PR, will have the changes pushed in by Thursday. Thanks again.